test/Microsoft.ML.Sweeper.Tests/Microsoft.ML.Sweeper.Tests.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <DefineConstants>CORECLR</DefineConstants>
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
><AllowUnsafeBlocks>true</AllowUnsafeBlocks> [](start = 4, length = 43)

Is `AllowUnsafeBlocks` being enabled only for the debug build intentional? That seems peculiar. I also don't see you're using unsafe blocks. How about removing this conditional block altogether? I don't see that it serves any purpose.

If I search our existing projects I see that perhaps `Microsoft.ML.Predictor.Tests.csproj` has the same problem. :(

Reply:
Hahahaha - I added it in as @sfilipi asked me to copy the Predictor tests - I will remove it - I do not use unsafe blocks

Reply:
If you like you like you could nuke it there while you're at it. If not, no worries.

---
In reply to: [196489732](https://github.com/dotnet/machinelearning/pull/368#discussion_r196489732) [](ancestors = 196489732)